src/libponyrt/actor/actor.c

```diff
@@ -673,6 +739,10 @@ PONY_API void pony_sendv(pony_ctx_t* ctx, pony_actor_t* to, pony_msg_t* first,
 
   pony_assert(well_formed_msg_chain(first, last));
 
+  // make sure we're not trying to send a message to an actor
+  // that is about to be destroyed
```

Comment:
To match with previous comment in the function can you make this a complete sentence with capitalization at the beginning and a period at the end?

Reply:
done